taxcalc/cli/tc.py

```diff
@@ -38,12 +44,12 @@ def main():
     parser.add_argument('--reform',
                         help=('REFORM is name of optional JSON reform file. '
                               'No --reform implies use of current-law '
-                              'policy.'),
+                              'policy (clp).'),
                         default=None)
     parser.add_argument('--assump',
                         help=('ASSUMP is name of optional JSON economic '
-                              'assumption file.  No --assump implies use of '
-                              'static analysis assumptions.'),
+                              'assumptions file.  No --assump implies use of '
+                              'standard (std) analysis assumptions.'),
```

Comment:
To me, "standard" implies that that a usage is 'approved' usage and that other uses are deviant. 

For example, consider two Tax-Calculator users on CNN presenting their results. One person could say, "I got X results assuming Y". The second person could say, "Y is just your assumption; when I use the standard model, I got Z." 

In this case, Z would sound more reasonable than X. 

The same dialogue could take place if the name were "base" instead of "standard", but I think the second guy would have a weaker point. 

Something even more neutral than "base" would be preferable, but I can't think of it.